src/client/visualizations/table/table.tsx

```diff
@@ -52,7 +52,7 @@ const SPACE_LEFT = 10;
 const SPACE_RIGHT = 10;
 const HIGHLIGHT_BUBBLE_V_OFFSET = -4;
 
-function formatSegment(value: any, timezone: Timezone): string {
+export function formatSegment(value: any, timezone: Timezone): string {
```

Comment:
if exported, move to formatter.ts in utils

Reply:
But then what is segment?

Should I change the name to `alsoFormatButDifferentlyThanFormatValue`? :D